pkg/smartcontract/callflag/call_flags.go

```diff
@@ -16,7 +22,93 @@ const (
 	NoneFlag CallFlag = 0
 )
 
+var flagString = map[CallFlag]string{
+	ReadStates:  "ReadStates",
+	WriteStates: "WriteStates",
+	AllowCall:   "AllowCall",
+	AllowNotify: "AllowNotify",
+	States:      "States",
+	ReadOnly:    "ReadOnly",
+	All:         "All",
+	NoneFlag:    "None",
+}
+
+var basicFlags = []CallFlag{ReadOnly, States, ReadStates, WriteStates, AllowCall, AllowNotify}
```

Comment:
The order is important for `String` and worth a comment.

Reply:
OK, added that.